start.sh

```diff
@@ -27,4 +27,7 @@ else
     source "${APP_DIR}/app_env"
 fi
 
-exec java ${JAVA_MEM_ARGS} -jar -Dserver.port="${PORT}" "${APP_DIR}/company-accounts.web.ch.gov.uk.jar"
\ No newline at end of file
+exec java ${JAVA_MEM_ARGS} -jar -Dserver.port="${PORT}" "${APP_DIR}/company-accounts.web.ch.gov.uk.jar"
+
+# debug settings
```

Comment:
Please can we revert this change?

Reply:
i will do it. We should revert it in the company-accounts api to be consistent